kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
maybe you might have rebased your orbital branch to `upstream/master` and then applied changes to what you saw at the time and that's why github correctly pointed to `master`